src/templates/utils.js

```diff
@@ -1,5 +1,25 @@
 import { LOCALE_CODE_KEY, LOCALE_FILE_KEY, MODULE_NAME } from './options'
 
+// Hiding template directives from eslint so that parsing doesn't break.
+/* <% if (options.langDir) { %> */
+/* <% const _srcDir = (isDev && (nuxtOptions.buildModules || []).includes('nuxt-vite')) ? '/@fs' + srcDir : '~' %> */
+async function loadFileModule (lang) {
+  let langFileModule
+  switch (lang) {
+    /* <% for (let locale of options.locales) { %> */
+    case '<%= locale.code %>':
+      langFileModule = await import(
+        /* webpackChunkName: "lang-[request]" */
+        /* webpackInclude: /\.(js|ts|json|ya?ml)$/ */
```

Comment:
If the file is explicitly specified now then I guess we don't need this directive anymore.